modules/persistent-storage-csi-sc-managing.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc
+//
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-sc-managing_{context}"]
+= Managing the default storage class using the web console
+
+.Prerequisites
+* Access to the {product-title} web console.
+
+* Access to the cluster with cluster-admin privileges.
+
+.Procedure
+
+To manage the default storage class using the web console:
+
+. Log in to the web console.
+
+. Click *Administration* > *CustomResourceDefinitions*.
+
+. On the *CustomResourceDefinitions* page, type `clustercsidriver` to find the `ClusterCSIDriver` object.
+
+. Click *ClusterCSIDriver*, and then click the *YAML* tab.
+
+. Change the `spec.storageClassState` field to `Managed`, `Unmanaged`, or `Removed`.
```

Comment:
Since the parameter spec.storageClassState is not visible when you first edit the object, it may be best to say add the field instead of change the field?
@dobsonj

Reply:
@dobsonj is that correct that you actually have to add the field? I can't get a cluster up to test this, but I don't remember clearly whether I had to do that or not.